lib/WebService/Stripe.pm

```diff
@@ -185,6 +191,14 @@ method get_bitcoin_receiver(Str $id, :$headers) {
     return $self->get( "/v1/bitcoin/receivers/$id", {}, headers => $headers );
 }
 
+method _stripe_opts_to_headers(HashRef $opts) {
+    my $headers = $opts->{'headers'} // {};
+    if ($opts->{'stripe_account'}) {
+        $headers->{'Stripe-Account'} = $opts->{'stripe_account'};
+    }
+    return $headers;
+}
```

Comment:
You mean adding an `apply_options` hook to WS::Client? :+1: I like that. But maybe with a better name.